.github/workflows/docs.yaml

```
@@ -0,0 +1,75 @@
+      - name: Start Uvicorn Server in Background
+        run: |
+          cd backend
+          pip install poetry
+          poetry install --no-root
+          nohup poetry run uvicorn main:app --host 0.0.0.0 --port 8000 &
+
+      - name: Wait for Server to Start
+        run: |
+          MAX_RETRIES=5
+          RETRY_INTERVAL=1
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)